File: Treinamento/C/Exercícios_listas/Questão02.c
#include <stdio.h>

int main() {
    double n_reais[10];

    for (int c = 0; c < 10; c++) {
        printf("Digite o %i° número: ", c + 1);
        scanf("%lf", &n_reais[c]);
    }

    for (int c = 0; c < 10; c++) {
        printf("%lf ", n_reais[c]);
    }
    printf("\n");

    for (int c = 9; c >= 0; c--) {
        printf("%lf ", n_reais[c]);
    }
    printf("\n");
}

File: Treinamento/Java/Exercícios_listas/Questão02.java
import java.util.Scanner;
public class Questão02 {
    public static void main(String[] args) {
        double vet_numeros[] = new double[10];
        Scanner s;
        s = new Scanner(System.in);

        for (int c = 0; c < 10; c++) {
            System.out.print("Digite o " + (c + 1) + " número: ");
            double n = s.nextDouble();
            vet_numeros[c] = n;
        }

        for (int c = 9; c >= 0; c--) {
            System.out.print(vet_numeros[c] + " ");
        }
        System.out.println();
    }
}

            
